backend/core/quivr_core/llm/llm_endpoint.py

```diff
@@ -19,13 +20,26 @@ def get_config(self):
     @classmethod
     def from_config(cls, config: LLMEndpointConfig = LLMEndpointConfig()):
         try:
-            from langchain_openai import ChatOpenAI
-
-            _llm = ChatOpenAI(
-                model=config.model,
-                api_key=SecretStr(config.llm_api_key) if config.llm_api_key else None,
-                base_url=config.llm_base_url,
-            )
+            if config.model.startswith("claude"):
```

Comment:
Just to reduce code boilerplate. if and only if the class have the exact params: 
```suggestion
if config.model.startswith("claude"): 
   from langchain_anthropic import ChatAnthropic
     cls = ChatAnthropic
else: 
        cls = ChaatOpenAI
```